packages/wdio-allure-reporter/tests/cucumber.suite.test.ts

```diff
@@ -261,14 +262,20 @@ describe('reporter option "useCucumberStepReporter" set to true', () => {
             reporter.onTestStart(cucumberHelper.testStart())
             reporter._consoleOutput = 'some console output'
             reporter.onTestSkip(cucumberHelper.testSkipped())
+
             const suiteResults: any = { tests: [cucumberHelper.testSkipped()] }
+
             reporter.onSuiteEnd(cucumberHelper.scenarioEnd(suiteResults))
             reporter.onSuiteEnd(cucumberHelper.featureEnd(suiteResults))
             reporter.onRunnerEnd(runnerEnd())
 
-            const results = getResults(outputDir)
+            const { results, containers } = getResults(outputDir)
+
+            allureResult = results[0]
+            allureContainer = containers[0]
+
             expect(results).toHaveLength(1)
-            allureXml = results[0]
+            expect(containers).toHaveLength(1)
```

Comment:
please assert length before you get value in line 275

Reply:
:+1: